app/models/user.rb

```diff
@@ -216,22 +216,10 @@ def miq_groups=(groups)
     self.current_group = groups.first if current_group.nil? || !groups.include?(current_group)
   end
 
-  def miq_group_ids
-    miq_groups.collect(&:id)
-  end
-
   def self.all_users_of_group(group)
     User.includes(:miq_groups).select { |u| u.miq_groups.include?(group) }
   end
 
-  def all_groups
-    miq_groups
-  end
```

Comment:
Wow, I hope git blame doesn't say me on this one.  Yeah, maybe it didn't exist in the rails version we added it.